src/content/self-hosted/install/releases.mdx

```diff
@@ -5,6 +5,36 @@ sidebar_label: Release Notes
 id: releases
 ---
 
+## 1.2.0
+
+### New Feature
+
+- Support for self-signed and private CA with Docker
+- Buildkit port is now configurable (default 443)
+- Add option for sharing namespace with users within organization
+- UI pagination support for GitHub branches list
+- Add permanently user deletion command
+- Allow to scale down to zero nodes if no user pods are running
+- Populate oauth errors to the user
+- Added new helm values to set PSP and AppArmor
+- Update okteto cli version to 2.9.0
+
+### Bug Fix
+
+- Don't show warning on k8s 1.23.x
+- Frontend nginx CVE-2022-40303/CVE-2022-40304
+- Fix loader icon regression
+- Fix in license display
```

Comment:
What was fixed?

Reply:
The seats https://github.com/okteto/app/issues/4799

Reply:
Maybe adding the `Fix` word is redundant due to we are in the `BugFix` section. In this case mentioned above, would it not be better: `Display license related information properly` ?